config/application.rb

```diff
@@ -86,8 +86,9 @@ class Application < Rails::Application
     # These defaults can be overridden per-environment if needed
     config.start_of_unique_links_only_intake = Time.find_zone('America/Los_Angeles').parse('2023-01-24 09:59:59')
     config.start_of_open_intake = Time.find_zone('America/Los_Angeles').parse('2023-01-31 09:59:59')
+    config.tax_deadline = Time.find_zone('America/New_York').parse('2023-04-18 23:59:59')
     config.end_of_intake = Time.find_zone('America/New_York').parse('2023-10-01 23:59:59')
-    config.end_of_login = Time.find_zone('America/New_York').parse('2023-10-15 23:59:00')
+    config.end_of_login = Time.find_zone('America/New_York').parse('2023-10-8 23:59:00')
```

Comment:
add another one for end of documents, different from end of login

Reply:
'end of docs' is consistently a week before end-of-login, so could be calc'd if that sounds fun